packages/react/src/MeasureReportDisplay/MeasureReportGroup/MeasureReportGroup.tsx

```diff
@@ -0,0 +1,127 @@
+import { Box, Flex, Group, Paper, RingProgress, Text, Title } from '@mantine/core';
+import { formatCodeableConcept } from '@medplum/core';
+import { Measure, MeasureReportGroup } from '@medplum/fhirtypes';
+import React from 'react';
+import { QuantityDisplay } from '../../QuantityDisplay/QuantityDisplay';
+
+interface MeasureGroupProps {
+  readonly group: MeasureReportGroup;
+}
+
+interface MeasureProps {
+  readonly measure: Measure;
+}
+
+export function MeasureReportDisplayGroup(props: MeasureGroupProps): JSX.Element | null {
+  const { group } = props;
+  return (
+    <Paper withBorder radius="md" p="xs" display="flex" sx={{ alignItems: 'center', justifyContent: 'center' }}>
+      <Group>
+        {group.measureScore && <MeasureScore group={group} />}
+        {!group.measureScore && <MeasureReportPopulation group={group} />}
+      </Group>
+    </Paper>
+  );
+}
+
+export function MeasureTitle(props: MeasureProps): JSX.Element {
+  const { measure } = props;
+  return (
+    <>
+      <Text fz="md" fw={500} mb={8}>
+        {measure.title}
+      </Text>
+      <Text fz="xs" c="dimmed" mb={8}>
+        {measure.subtitle}
+      </Text>
+    </>
+  );
+}
+
+function MeasureReportPopulation(props: MeasureGroupProps): JSX.Element {
+  const { group } = props;
+  const populations = group.population;
+  const numerator = populations?.find((p: any) => formatCodeableConcept(p.code) === 'numerator');
+  const denominator = populations?.find((p: any) => formatCodeableConcept(p.code) === 'denominator');
+
+  const numeratorCount = numerator?.count;
+  const denominatorCount = denominator?.count;
+
+  if (!numeratorCount || !denominatorCount) {
```

Comment:
Beware the case where `numeratorCount === 0`.  We should explicitly check for `numeratorCount === undefined` here.

The case where `denominatorCount === 0` is slightly different, and may require a different special case?  It's not really "Insufficient Data", but we should guard against divide-by-zero errors.

Reply:
I added 'Not Applicable'
<img width="254" alt="Screenshot 2023-11-07 at 1 28 07 PM" src="https://github.com/medplum/medplum/assets/6913745/566f7486-7ae3-4615-93ba-f47508bbb943">